handout/tools.py

```diff
@@ -12,3 +15,17 @@ def strip_empty_lines(lines):
     output.append(line)
   lines = reversed(output)
   return list(lines)
+
+
+def get_logger():
+  logger = logging.getLogger('handout')
+  logger.setLevel(logging.INFO) # A user can change this level later.
```

Comment:
Regarding your comment for user levels, the user can change it globally by running this line, for example:
```
logger.setLevel(logging.DEBUG)
```

Also we actually do not provide any other logging messages than info().

Reply:
Well, actually he wouldn't change the log level for that particular logger... need a rethink.